easybuild/easyconfigs/c/CP2K/CP2K-6.1-CrayGNU-18.08-cuda-9.1.eb

```diff
@@ -6,12 +6,6 @@ version = '6.1'
 cudaversion = '9.1'
 versionsuffix = '-cuda-%s' % cudaversion
 
-py_maj_ver = '2'
```

Comment:
We need to restore the `Python` variables in order to comply with the standard template described in the Wiki: I'll push a new version of the recipe for `CP2K`